### PR TITLE
Updates for rococo

### PR DESCRIPTION
**Pull request by stanly-johnson**
- Minor fixes and cleanup
- Setup up testnet spec for rococo

### CODE REVIEW

File: node/parachain/src/command.rs [hunk 1]
```diff
@@ -56,7 +43,7 @@ impl SubstrateCli for Cli {
 
     fn description() -> String {
         format!(
-            "Polkadot collator\n\nThe command-line arguments provided first will be \
+            "Parachain Collator Template\n\nThe command-line arguments provided first will be \
```
Comment (olanod) [inline node/parachain/src/command.rs:46]:
```suggestion
            "Parachain Collator\n\nThe command-line arguments provided first will be \
```

File: node/parachain/src/command.rs [hunk 2]
```diff
@@ -69,47 +56,45 @@ impl SubstrateCli for Cli {
...
     }
 
     fn impl_version() -> String {
         env!("SUBSTRATE_CLI_IMPL_VERSION").into()
     }
 
     fn description() -> String {
-        format!(
-            "Polkadot collator\n\nThe command-line arguments provided first will be \
+        "Parachain Collator Template\n\nThe command-line arguments provided first will be \
```
Comment (olanod) [inline node/parachain/src/command.rs:85]:
```suggestion
        "Parachain Collator\n\nThe command-line arguments provided first will be \
```

File: scripts/aura_key.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (olanod) [inline scripts/aura_key.json:1]: mmm having this versioned controlled might make us submit keys by mistake. perhaps add it as an example and the script not be depending on this specific file being there but just expect a json file as argument

**Review by olanod — COMMENTED** [review]
With the new Aura stuff have you tested running with one collator? how many collators are we actually going to need as minimum?

**Comment by stanly-johnson** [comment]
> With the new Aura stuff have you tested running with one collator? how many collators are we actually going to need as minimum?

One collator works, either one of the collators have to be online.
👇 with only one collator alive
```
2021-06-02 23:12:12.087  INFO tokio-runtime-worker sc_basic_authorship::basic_authorship: [Parachain] 🙌 Starting consensus session on top of parent 0x7333fbfbbe0aab67d9118c536c08b615815a6843917f59d91191d4cca3dbdd93    
2021-06-02 23:12:12.103  INFO tokio-runtime-worker sc_basic_authorship::basic_authorship: [Parachain] 🎁 Prepared block for proposing at 28 [hash: 0xa00d46b26faa6c410d3409e1115adf26dd58c411a92684739157e0b21bde6620; parent_hash: 0x7333…dd93; extrinsics (2): [0x7ec5…efa8, 0x8691…47d4]]    
2021-06-02 23:12:12.104  INFO tokio-runtime-worker aura: [Parachain] 🔖 Pre-sealed block for proposal at 28. Hash now 0xa063aa33ecc8ff8ac85c5ffcb54145890b1e6491e936944a2cc82f286f04848e, previously 0xa00d46b26faa6c410d3409e1115adf26dd58c411a92684739157e0b21bde6620.    
2021-06-02 23:12:12.104  INFO tokio-runtime-worker substrate: [Parachain] ✨ Imported #28 (0xa063…848e)    
2021-06-02 23:12:12.107  INFO tokio-runtime-worker cumulus-collator: [Parachain] Produced proof-of-validity candidate. block_hash=0xa063aa33ecc8ff8ac85c5ffcb54145890b1e6491e936944a2cc82f286f04848e
2021-06-02 23:12:13.774  INFO tokio-runtime-worker substrate: [Parachain] 💤 Idle (0 peers), best: #27 (0x7333…dd93), finalized #27 (0x7333…dd93), ⬇ 0 ⬆ 0    
2021-06-02 23:12:13.775  INFO tokio-runtime-worker substrate: [Relaychain] 💤 Idle (2 peers), best: #149 (0x272d…413b), finalized #146 (0xc2af…27d1), ⬇ 1.4kiB/s ⬆ 0.4kiB/s    

```

**Review by olanod — APPROVED** [review]

**Pull request merged**